include/sancus_support/sancus_step.h

```diff
@@ -3,50 +3,101 @@
 #include <msp430.h>
 #include "timer.h"
 #include <stdint.h>
+#include <sancus/sm_support.h>
 
-#define HW_IRQ_LATENCY          34
-#define ISR_STACK_SIZE          512
-#define INIT_LATENCY            42
 
-void print_latency(void);
+/* ======== SANCUS STEP DBG CONSTANTS ======== */
+#define RETI_LENGTH (0x5)
+#define JMP_LENGTH (0x2)
+#define RET_LENGTH (0x3)
 
-uint16_t __ss_isr_stack[ISR_STACK_SIZE];
-void*    __ss_isr_sp;
-void*    __ss_isr_reti_addr;
-volatile int      __ss_isr_tar_entry;
+#define EXTRA_DELAY (0x2)
 
-/*
- * The first latency will be useless, but this is ok for now.
- * We can fix this by adjusting the INIT_LATENCY definition.
- */
-#define SANCUS_STEP_INIT                    \
-    __asm__("dint\n\t");                    \
-    TACTL = TACTL_DISABLE;                  \
-    TACCR0 = INIT_LATENCY;                  \
-    /* source mclk, up mode */              \
-    TACTL = TACTL_ENABLE;
+#define INIT_LATENCY 1
+
+void sancus_step_print_latency(void);
+int sancus_step_get_latency(void);
+
+// sancus step interface
+void sancus_step_start(void);
+void sancus_step_init(void);
+void sancus_step_end(void);
+
+// sancus step configuration parameters
```

Comment:
to avoid unintended name collisions in linked code, I propose you stick consistently to the `__ss` name space for all global variable names in the sancus-step library as you already did in some places?